x-pack/plugins/apm/public/components/shared/links/apm/service_link/service_max_groups_message.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export const OTHER_SERVICE_NAME = '_other';
+
+export function ServiceMaxGroupsMessage({
+  remainingServices = 1,
+}: {
+  remainingServices?: number;
```

Comment:
Let's make this required on the server, where it should default to 0.

Reply:
@sqren Initially i started with the same approach, but the problem here is, this component is used inside `ServiceLink` component and `ServiceLink` component is used in multiple places like - `Storage Explorer` , `Dependencies Detail Table`, `Flyout Top Level`, etc.

Only place where it will get this value will be in the `Service Inventory` component, hence had to make it optional here